route/task.js

```diff
@@ -100,6 +100,7 @@ module.exports = function (app) {
 					query: {},
 					payload: {
 						name: Hapi.types.String().required(),
+						timeout: Hapi.types.Number().integer().required(),
```

Comment:
I don't think this should be a required property, as it would break backwards-compatibility